signer/keydbstore/keydbstore.go

```diff
@@ -179,7 +179,7 @@ func (s *KeyDBStore) RotateKeyPassphrase(name, newPassphraseAlias string) error
 	// Retrieve the GORM private key from the database
 	dbPrivateKey := GormPrivateKey{}
 	if s.db.Where(&GormPrivateKey{KeyID: name}).First(&dbPrivateKey).RecordNotFound() {
```

Comment:
just for my sanity, can we rename the `name` parameter to `keyID` because that's what it must be based on this query vs other queries on the `GormPrivateKey` type.